Source/SupportingFiles/Shared.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Shared.swift
+//  WatsonDeveloperCloud
+//
+//  Created by Anthony Oliveri on 9/4/18.
+//  Copyright © 2018 Glenn R. Fisher. All rights reserved.
+//
+
+import Foundation
+import RestKit
+
+
+/// Contains functionality and information common to all of the services
+struct Shared {
```

Comment:
On further consideration, I think "Common" might be a better name for this struct. I like this better than "Shared" because is seems more passive.

Reply:
I picked "Shared" because it's a common (no pun intended) term used in Swift, like in `URLSession.shared`. "Common" is a good name too, so I can switch it to that if you feel that's more appropriate.